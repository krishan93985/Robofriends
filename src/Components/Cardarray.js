import React from 'react';
import Card from "./Card";

const Cardarray = ({robots}) => {
    return(
    <div>
        {robots.map((user,i) => 
        <Card key={i} id={user.id} name={user.name} username={user.username} email={user.email}/>
        )
        }
    </div>
    );
}

export default Cardarray;