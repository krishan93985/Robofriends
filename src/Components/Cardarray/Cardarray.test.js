import React from 'react';
import Cardarray from './Cardarray';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

it('is Testing Card component', () => {
    //mocking robots props
    const mockRobots = [
        {
            id:"1",
            name:"trish",
            username:"trish12345",
            email:"trish@gmail.com"
        }
    ]
    const wrapper = shallow(<Cardarray robots={mockRobots} />);
    expect(toJson(wrapper)).toMatchSnapshot();
})