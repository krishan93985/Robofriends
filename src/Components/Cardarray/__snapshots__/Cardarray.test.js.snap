// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is Testing Card component 1`] = `
<div>
  <Card
    email="trish@gmail.com"
    id="1"
    key="0"
    name="trish"
    username="trish12345"
  />
</div>
`;
